cvat-ui/src/actions/tasks-actions.ts

```diff
@@ -169,7 +169,7 @@ ThunkAction<Promise<void>, {}, {}, AnyAction> {
     return async (dispatch: ActionCreator<Dispatch>): Promise<void> => {
         try {
             dispatch(dumpAnnotation(task, dumper));
-            const url = await task.annotations.dump(task.name, dumper);
+            const url = await task.annotations.dump('', dumper);
```

Comment:
Why have you decided left name here as the first argument if it doesn't used anywhere?

Reply:
Argument is made optional